lib/widget/input/form_helper.dart

```diff
@@ -74,6 +76,8 @@ class FormFieldController extends WidgetController {
           focusedBorderColor = Utils.getColor(color),
       'focusedErrorBorderColor': (color) =>
           focusedErrorBorderColor = Utils.getColor(color),
+      'hintStyle': (style) => hintStyle = Utils.getTextStyle(style),
```

Comment:
You added these to form helper, so all form inputs will have it. Please move the schema to all form input widget, and also confirm all those form inputs will work.
If you just want the TextInput for now, move the variables to TextInput controller.